crypto/hd/algo.go

```diff
@@ -4,7 +4,9 @@ import (
 	bip39 "github.com/cosmos/go-bip39"
 
 	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/sr25519"
 	"github.com/cosmos/cosmos-sdk/crypto/types"
+	substratebip39 "github.com/sikkatech/go-substrate-bip39"
```

Comment:
```suggestion
	substratebip39 "github.com/sikkatech/go-substrate-bip39"
```

I believe should be hosted in a cosmos/ repository